src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java

```diff
@@ -51,6 +51,10 @@ public KildaConfiguration getKildaConfiguration() {
      */
     public KildaConfiguration updateKildaConfiguration(KildaConfiguration kildaConfiguration) {
         log.info("Process kilda config update - config: {}", kildaConfiguration);
+        if (Boolean.FALSE.equals(kildaConfiguration.getUseMultiTable())) {
```

Comment:
Just curious, what is this trick with `Boolean..equals()` needed for?

Reply:
kildaConfiguration.getUseMultiTable() can be `null` and it's okay. 
But I need to check that the value is not false. 
If I will use `if (!kildaConfiguration.getUseMultiTable())` I can get NPE